### ISSUE TITLE
`JWT_ALLOW_ANY_CLASSES` usage example

### ISSUE THREAD

**Issue by coler-j** [flavors/django-graphql-jwt#218]
Is there a usage example of `JWT_ALLOW_ANY_CLASSES`? 

From my interpretation it seems like it should be a list of query or mutation classes where authentication will not be applied. However... it seems like authentication needs to be manually applied at a class function level anyways (i.e. with `loginRequired` decorator). 

So what is the point of `JWT_ALLOW_ANY_CLASSES` if authentication is manually applied? Wouldn't you just _not_ add a decorator to the function you don't want authentication at?

Or is there a global way to apply authentication that is missing from the docs?

**Comment by vberlier** [comment]
Agreed. I've been wondering the same thing. The documentation is a bit unclear on this.

**Comment by yashttm** [comment]
Bumping this because just pondering on the same issue.

**Comment by andrewtmendoza** [comment]
I'm wondering the same thing :)

**Comment by nielsvdnieuwendijk** [comment]
I'm wondering the same thing....

**Comment by LinnaViljami** [comment]
I have same issue
